install.md

```diff
@@ -200,6 +200,30 @@ mkdir -p /usr/share/keyrings
 curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/Release.key | gpg --dearmor -o /usr/share/keyrings/libcontainers-archive-keyring.gpg
 curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/$VERSION/$OS/Release.key | gpg --dearmor -o /usr/share/keyrings/libcontainers-crio-archive-keyring.gpg
 
+apt-get update
+apt-get install cri-o cri-o-runc
+```
+
+If apt doesn't allow redirection from 'https' -> 'http' mirror then modify the protocol as follows
+
+```shell
+
+# if something similar to the below log happens...
+
+Redirection from https to 'http://mirror.blablah.com/opensuse/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:versionblah/osblah' is forbidden
```

Comment:
I'd say we change `versionblah` and `osblah` to something without `blah` :)

Reply:
great idea, I'll fix them